release-please-config.json

```diff
@@ -12,6 +12,16 @@
         "README.md"
       ]
     },
+    "providers/flagsmith": {
+      "package-name": "dev.openfeature.contrib.providers.flagsmith",
+      "release-type": "simple",
+      "bump-minor-pre-major": true,
+      "bump-patch-for-minor-pre-major": true,
+      "versioning": "default",
+      "extra-files": [
+        "pom.xml"
```

Comment:
You'll also need to include `README.md` to the `extra-files` list. This will configure Release Please to automatically bump the version in the readme as part of a release PR.

Reply:
I did this.